draft-ietf-dnsop-svcb-https.md

```diff
@@ -642,6 +642,10 @@ resolution procedure, as if processing a query for that RRSet.
 This includes following any aliases that the resolver would ordinarily
 follow (e.g. CNAME, DNAME {{!DNAME=RFC6672}}).
 
+Recursive resolvers MUST implement reasonable loop prevention,
```

Comment:
The AliasForm section already says "To avoid unbounded alias chains, clients and recursive resolvers MUST impose a limit on the total number of SVCB aliases they will follow".  I don't understand whether this requirement is equivalent or not.

Reply:
To clarify, I think we need to answer whether "a limit on the total number of SVCB aliases" counts as "reasonable loop prevention".  If it does, then this line is redundant.  If it doesn't, then the intent here is to require a specific loop-detection behavior (before hitting the alias limit), which I think would need to be stated more explicitly.

Reply:
Chrome intends an alias limit small enough that extra loop detection beyond that would just be silly.  Please don't make me implement silly loop detection.